.rubocop_todo.yml

```diff
@@ -36,13 +36,11 @@ Layout/CommentIndentation:
   Exclude:
     - 'lib/dogapi/event.rb'
 
-# Offense count: 3
+# Offense count: 1
 # Cop supports --auto-correct.
 Layout/EmptyLineAfterMagicComment:
   Exclude:
     - 'dogapi.gemspec'
-    - 'spec/integration/comment_spec.rb'
```

Comment:
Are these no longer needed?

Reply:
ran the rubocop autogenerate command `rubocop --auto-gen-config` with the ruby version that's used for linting so I think this should be good